var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -17,58 +19,129 @@ class UfsWeatherModel(CMakePackage):
     url = "https://github.com/ufs-community/ufs-weather-model/archive/refs/tags/ufs-v1.1.0.tar.gz"
     git = "https://github.com/ufs-community/ufs-weather-model.git"
 
-    maintainers = ["t-brown"]
+    maintainers = ["t-brown", "AlexanderRichert-NOAA"]
 
+    version("develop", branch="develop", submodules=True, commit="ea0b6e4")
     version("2.0.0", tag="ufs-v2.0.0", submodules=True)
     version("1.1.0", tag="ufs-v1.1.0", submodules=True)
 
+    variant("mpi", default=True, description="Enable MPI")
     variant(
         "32bit", default=True, description="Enable 32-bit single precision arithmetic in dycore"
     )
-    variant("avx2", default=False, description="Enable AVX2 instructions")
     variant(
-        "ccpp", default=True, description="Enable the Common Community Physics Package (CCPP))"
+        "ccpp_32bit",
+        default=False,
+        description="Enable CCPP_32BIT (single precision arithmetic in slow physics)",
+    )
+    variant("avx2", default=True, description="Enable AVX2 instruction set")
+    variant(
+        "simdmultiarch", default=False, description="Enable multi-target SIMD instruction sets"
+    )
+    variant("debug", default=False, description="Enable DEBUG mode", when="@develop")
+    variant(
+        "debug_linkmpi",
+        default=True,
+        description="Enable linkmpi option when DEBUG mode is on",
+        when="@develop",
+    )
+    variant("inline_post", default=False, description="Enable inline post")
+    variant("multi_gases", default=False, description="Enable multi gases in physics routines")
+    variant("moving_nest", default=False, description="Enable moving nest code", when="@develop")
+    variant("openmp", default=True, description="Enable OpenMP")
+    variant("parallel_netcdf", default=True, description="Enable parallel NetCDF")
+    variant(
```

Comment:
I don't know if this option works at all, it's certainly not the way we integrate the UFS with JEDI here: https://github.com/JCSDA/ufs-bundle/pull/9

@mark-a-potts @aerorahul Can you provide more information on this option?